docs/internals/deprecation.txt

```diff
@@ -120,6 +120,8 @@ details on these changes.
   ``Expression.asc()`` and ``Expression.desc()`` methods, and the ``OrderBy``
   expression will be removed.
 
+* ``User.objects`` will no longer have a ``make_random_password`` method.
```

Comment:
This should be in the [5.1 sectiom](https://docs.djangoproject.com/en/dev/internals/deprecation/#deprecation-removed-in-5-1).